tools/flavor-convert/go.mod

```diff
@@ -0,0 +1,13 @@
+module github.com/flavor-convert
+
+require (
+	github.com/antchfx/jsonquery v1.1.4
+	github.com/google/uuid v1.2.0
+	github.com/intel-secl/intel-secl/v3 v3.4.0
+	github.com/stretchr/testify v1.7.0 // indirect
+	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
+	github.com/vmware/govmomi v0.22.2
+)
+
+replace github.com/intel-secl/intel-secl/v3 => github.com/isteffyx/intel-secl/v3 v3.4.0
```

Comment:
This tool should work with the latest Steffy's fork?

Reply:
Here in flavor convert tool, we are using flavor template and new flavor part struct from intel-secl repo. But, Since the new flavor part changes are not present in intel-secl master branch, we have pushed the flavor creation/verification changes in our own repo (isteffyx/intel-secl) and replaced the intel-secl path with this repo path in go mod file. Once the flavor creation/verification changes are merged with intel-secl master branch, we will remove this path from go mod file.